docs/sphinx/using/hardware.rst

```diff
@@ -0,0 +1,202 @@
+CUDA Quantum Hardware Backends
+*********************************
+
+CUDA Quantum supports submission to a set of hardware providers. 
+To submit to a hardware backend, you need an account with the respective provider.
+
+IonQ
+==================================
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA Quantum may access the `IonQ Quantum Cloud
+<https://cloud.ionq.com/>`_ from either C++ or Python. Simply generate
+an API key from your `IonQ account <https://cloud.ionq.com/>`_ and export
+it as an environment variable:
+
+.. code:: bash
+
+  export IONQ_API_KEY="ionq_generated_api_key"
+
+
+C++
+```
+
+For developers in C++, you can indicate to ``nvq++`` that your quantum
+kernels will be executed in the IonQ Cloud via the ``--target`` flag.
+
+.. code:: bash
+
+    nvq++ --target ionq src.cpp ...
+
+This will take the API key and handle all authentication with, and submission to,
+the IonQ QPU.
+
+Note: A "target" in :code:`cudaq` refers to a quantum compute provider, such as :code:`ionq`.
+However, IonQ's documentation uses the term "target" to refer to specific QPU's themselves.
+
+Specifying the QPU
+'''''''''''''''''''
+
+At this time, programmer control over the IonQ QPU is under construction in ``nvq++``.
```

Comment:
This needs to be changed. There is a way to define what machine to submit to, and we need to document it. Otherwise the backends are not usable (we can't ask people to guess).

Reply:
Would you mind pointing me to where this exists? As of Tuesday I was told that it's not currently hooked up to nvq++ and is only supported in Python

Reply:
Wow, this pops up really late! I updated the docs and the ionq config to speculatively add it, but it will first need to be properly confirmed that it works.